tensorflow/python/ops/rnn.py

```diff
@@ -73,6 +73,7 @@ def rnn(cell, inputs, initial_state=None, dtype=None,
   """Creates a recurrent neural network specified by RNNCell `cell`.
 
   The simplest form of RNN network generated is:
+
```

Comment:
There's a `+` there. It's not an empty line. The reason why these empty lines work is because the following block is indented. It is much preferable to use the triple-backtick syntax.